adding gemini to the supported apps by the plugin

Samir-atra opened the pull request:
catching the conversations from the gemini.google.com and documenting them, in addition to the previously supported models and websites.

shachardon commented on README.md:
```diff
@@ -1,2 +1,85 @@
+
+## Usage
+
+The ShareLM plugin in action. You can use the plugin popup to go over your collected conversations from the last 24 hours and rate them. You can also use the popup to delete conversations that you don’t want to share before they are pushed to the database.
```
remove the "The ShareLM plugin in action." The rest if fine

Samir-atra commented on extension/index_frame.js:
```diff
@@ -594,40 +616,32 @@ function init() {
-          if (i < cur_bot_msgs.length - 1 || cur_user_msgs[i] !== new_user_msgs[i]) { // if this is only the last message that changed, it's not a new conversation
-            new_conversation = true;
-          }
+          new_conversation = true; // Force a new conversation for every change
```
Okay I will edit

Samir-atra commented on extension/index_frame.js:
```diff
@@ -715,52 +727,90 @@ function init() {
+    function queryAndUpdateConversations(user_selector, bot_selector, sub_user_selector, sub_bot_selector, strategy = "css") {
     if (!shouldShare || !age_verified) {
-      console.log("Sharing is disables, not updating conversation");
+      console.log("Sharing is disabled, not updating conversation");
```
Thanks

Samir-atra commented on extension/index_frame.js:
```diff
@@ -715,52 +727,90 @@ function init() {
         });
-      });
+        break;
+      case "text":
```
I am not sure, it is the AI, I was vibe coding and will check what it is exactly.

Samir-atra replied:
You are looking to revert the changes and delete the text case?
It was for debugging at the time of development, and not necessary now, but it might help later. So, should I cancel the switch?

Samir-atra replied:
You are looking to revert the changes and delete the text case?
It was for debugging at the time of development, and not necessary now, but it might help later. So, should I cancel the switch?

Samir-atra commented:
I added the changes for the one plugin list element per conversation, and found some inconsistency between the banner behaviour when using Gemini and Claude, so I fixed this, too.

Samir-atra commented:
The inconsistency was that the banner was not showing all the time.
Now I removed the switch and the text case, and fixed the banner.
